### ISSUE TITLE
A11y_SQLAzureDataStudio_DeployDacpac_DeployDacpacSettings_Screenreader: Non actionable column headers 'Settings' and 'value' is announced as 'Clickable' by screenreader.

### ISSUE THREAD

**Issue by shweta255** [microsoft/azuredatastudio#9324]
[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

GitHubTags:#A11y_SQLAzureDataStudioBenchmark;#A11yMAS;#A11ySev2;#A11yTCS;#SQL Azure Data Studio;#Benchmark;#DesktopApp;#Win32; #MAS1.3.1;#Screenreader;

**Environment Details:**
Version: 1.16.0-insider
Commit: 506c6a5
Date: 2020-02-17T05:43:18.399Z
VS Code: 1.42.0
Electron: 7.1.11
Chrome: 78.0.3904.130
Node.js: 12.8.1
V8: 7.8.279.23-electron.0
OS: Windows_NT x64 10.0.18363
Additional Details:
MAS References: MAS1.3.1
NVDA Version: 2019.2

**Repro Steps:**
1. Launch Azure Data Studio application.
2. Connect to server with windows credentials.
3. Navigate to Extensions from left menu.
4. Search & install Dacpac extension.
5. Navigate to Connections from left menu.
6. Expand the server connected in step 2.
7. Right click an on prem database and select Data-tier application wizard from Context menu.
8. Select second radio button.
9. Navigate till next button & hit enter.
10. Provide all the details & hit enter.
11. Launch NVDA screenreader
12. Navigate to column headers 'Settings' and 'value' using down arrow key

**Actual:**
While NVDA is on and navigated to column headers 'Settings' and 'value' is announced as 'Clickable', screenreader announces as Clickable.

**Same issue is observed in Extract Dacpac Settings screen:**Navigation:
 1. Launch Azure Data Studio application.
2. Connect to server with windows credentials.
3. Navigate to Extensions from left menu.
4. Search & install Dacpac extension.
5. Navigate to Connections from left menu.
6. Expand the server connected in step 2.
7. Right click an on prem database and select Data-tier application wizard from Context menu.
8. Select second radio button.
9. Navigate till next button & hit enter.

**Expected:**
While NVDA is on and navigated to column headers 'Settings' and 'value', screenreader should not announce as 'Clickable' as column headers are not actionable.

**User Impact:**
Screen reader users will face difficulty as screenreader announces as 'Clickable' for non actionable column headers.

**Attachment link for Reference:**
[9324_A11y_SQLAzureDataStudio_DeployDacpac_DeployDacpacSettings_Screenreader.zip](https://github.com/microsoft/azuredatastudio/files/4274809/9324_A11y_SQLAzureDataStudio_DeployDacpac_DeployDacpacSettings_Screenreader.zip)

**Comment by alanrenmsft** [comment]
need to remove the click handler if the header is not clickable.

**Comment by kisantia** [comment]
Same as https://github.com/Microsoft/azuredatastudio/issues/6662 which was approved for exclusion

**Comment by BhanuPrakashGit** [comment]
@alanrenmsft Could you please the reopened the bug as it should be verified the Trusted Tested.
Please provide the latest environment details where we can verify this and take action accordingly.

GithubTags:#Regressed:06-02-23;

**Comment by alanrenmsft** [comment]
reopen as requested by testing team. please verify.

**Comment by alanrenmsft** [comment]
hmm, I couldn't find the email thread that requested me to reopen this.

**Comment by BhanuPrakashGit** [comment]
@alanrenmsft Tried to Verfied the bug, but the screen, given below, keep loading. Could you please look into this.
**GithubTags:**#Regressed:02-17-23;
![image](https://user-images.githubusercontent.com/104351566/219649936-d8cf9451-c5c5-4d0a-9fe4-e2fafd77ea11.png)

**Comment by alanrenmsft** [comment]
@kisantia , could you please take a look?

**Comment by kisantia** [comment]
@BhanuPrakashGit I shared a dacpac and the database for testing over email. Let me know if you need anything else

**Comment by mstechie** [comment]
GitHubTags:#SQLADS-Win32-Feb2020;

**Comment by BhanuPrakashGit** [comment]
**GithubTags:**#Closed; #Regressed:03-13-23;
Verified the bug in the environment: 
Version: 1.43.0-insider (system setup)
Commit: c163830bc4611af6c01458e7f5bd8f2546c4f523
Date: 2023-03-10T06:34:56.782Z
VS Code: 1.67.0
Electron: 19.1.8
Chromium: 102.0.5005.167
Node.js: 16.14.2
V8: 10.2.154.15-electron.0
OS: Windows_NT x64 10.0.22621

Bug is fixed and does not repro. 

@kisantia Please close this bug as it is fixed. PFA for reference

[9324_Closed.zip](https://github.com/microsoft/azuredatastudio/files/10954782/9324_Closed.zip)
